frontend/views/containers/access/LoginForm.vue

```diff
@@ -67,6 +70,8 @@ export default {
   },
   methods: {
     async login () {
+      if (this.ephemeral.isSubmitting) { return }
+      this.ephemeral.isSubmitting = true
```

Comment:
Instead of introducing a variable called `isSubmitting`, is there a way to simply disable the submit button using something like:

```pug
:disabled='$v.form.$invalid'
```

?

Reply:
Hum, but we already do that. (and [I don't agree](https://axesslab.com/disabled-buttons-suck/) with it)

Reply:
I mean disabling the button after it is pressed. I skimmed that link just now and it seems to be referring to buttons that are disabled initially, rather than buttons that are initially enabled, and disabled when clicked, to indicate that an action has taken place, and that the user needs to wait for it to complete before clicking it again.

The concern in that link is that clicking the button does nothing. And yet that is exactly what would happen in this scenario above, where the button is enabled, but clicking it multiple times does nothing if it takes a while to perform the submit action.

Reply:
> The concern in that link is that clicking the button does nothing. 

That's why we need visual loading states on this scenarios as per #802.

So, I'll change to `:disabled='$v.form.$invalid || ephemeral.isSubmitting'` does that sound ok to you?

Reply:
But still, if we press "Enter" instead of clicking the button, it will attempt to submit the formagain, so we still need the line `if (this.ephemeral.isSubmitting) { return }`. In other words, doing what I suggested on last comment is not useful.

Reply:
I would leave it as it is and wait for @mmbotelho  solution on #802.

Reply:
OK, all of that sounds reasonable 👍

Reply:
Cool! Resolving this conversation, since there's nothing left to change.